docs/blocks-storage/querier.md

```diff
@@ -1544,4 +1544,34 @@ blocks_storage:
     # [EXPERIMENTAL] True to enable native histogram.
     # CLI flag: -blocks-storage.tsdb.enable-native-histograms
     [enable_native_histograms: <boolean> | default = false]
+
+    # [EXPERIMENTAL] If enabled, ingesters will cache expanded postings when
+    # querying blocks. Caching can be configured separately for the head and
+    # compacted blocks.
```

Comment:
It is not clear to me if this cache is global or per block for both head and disk. Maybe I missed something. Do we need to document that better?

Reply:
fair.. added it back! :P

Reply:
Yes. Looks like it is global?

Reply:
Yeah.. its global.

We create a cache for head and for the disk. And the disk one is "global". 

Lemme try to clarify a bit.

Reply:
Done!